### PR TITLE
Update fr_FR.lang

### PR DESCRIPTION
**Pull request by Trounk**
So like this ? :)

### CODE REVIEW

**Comment by LovelyFlare** [comment]
Yes, like that.

File: resources/assets/tconstruct/lang/fr_FR.lang [hunk 1]
```diff
@@ -862,8 +938,7 @@ option.tconstruct.tank=Réservoir
 option.tconstruct.casting=Moulage
 option.tconstruct.progress=Progression de la recette
 
+
 # Config
-config.synced.restart=Votre configuration a été mise à jour pour correspondre au serveur. Relancez Minecraft.
+config.synced.restart=votre configuration a été mise à jour pour correspondre au serveur. Relancez Minecraft.
```
Comment (LovelyFlare) [inline resources/assets/tconstruct/lang/fr_FR.lang]: Why is this decapitalized?  This is the message the game gives in chat when the configs sync on server join.

Comment (Trounk) [inline resources/assets/tconstruct/lang/fr_FR.lang]: Hum hum, its a good question, maybe an error of copy-paste, I check :D

**Comment by bonii-xx** [comment]
Great, thanks! I'll merge it in together with the russian version when it's there. If you find something else amiss, you can fix it that way.

**Pull request merged**